Automattic/wp-calypso
# Add bilmur to Calypso

In order to fill in some of the gaps on the Calypso RUM library, compare data between implementations, and obtain a better overall picture across `wordpress.com`, we're adding the `bilmur` RUM library to Calypso, in production.

This will capture performance stats on 100% of initial page loads (not in-SPA navigations). Due to the way that `bilmur` has been built, the numbers for LCP and CLS probably won't make a ton of sense on a SPA like Calypso, but they'll still provide some useful indications for initial page loads.

#### Changes proposed in this Pull Request

* Add `bilmur` to production Calypso

#### Testing instructions

* Build and load Calypso as-is
* Ensure that the `bilmur` script isn't loaded
* Build and load Calypso with `ENABLE_FEATURES=bilmur-script`
* Ensure that the `bilmur` script is loaded
* Ensure that a request is made to `boom.gif` with `bilmur=1` and the various metrics
